libraries/AP_Baro/AP_Baro.h

```diff
@@ -12,6 +13,11 @@
 // multiple sensor instances
 #define BARO_MAX_DRIVERS 3
 
+// maximum vertical accuracy from GPS that will be accepted as baro correction (meters)
+#define BARO_MAX_GPS_ACCURACY 2
```

Comment:
Nitpick, I'd think of this as the minimum GPS accuracy required to do the blending. Not sure it's worth obsessing about. This should probably be declared as 2.0f given that it's only used in a float comparison, but the compiler should be fixing it.